src/Fmt.ml

```diff
@@ -164,13 +164,16 @@ let box_depth_colors = [|32; 33; 94; 31; 35; 36|]
 let box_depth_color () =
   box_depth_colors.(!box_depth % Array.length box_depth_colors)
 
-let debug_box_open box_kind n fs =
+let debug_box_open ?name box_kind n fs =
   if !box_debug_enabled then (
     let openning =
       if n = 0 then box_kind else Format.sprintf "%s<%d" box_kind n
     in
+    let name =
+      Option.value_map name ~f:(fun s -> Format.sprintf "<%s>" s) ~default:""
+    in
     pp_color_k (box_depth_color ())
-      (fun fs -> Format.fprintf fs "@<0>[@<0>%s@<0>>" openning)
+      (fun fs -> Format.fprintf fs "@<0>[@<0>%s@<0>%s>" openning name)
```

Comment:
```suggestion
      (fun fs -> Format.fprintf fs "@<0>[@<0>%s@<0>%s@<0>>" openning name)
```
Otherwise the closing `>` will be counted and formatting may be different due to the margin.

Reply:
Fixed